stripe/src/main/java/com/stripe/android/Stripe.java

```diff
@@ -66,21 +67,28 @@ protected void onPostExecute(ResponseWrapper result) {
         }
     };
 
+    private Context mContext;
     private StripeApiHandler.LoggingResponseListener mLoggingResponseListener;
     private String mDefaultPublishableKey;
 
     /**
-     * A blank constructor to set the key later.
+     * A constructor with only context, to set the key later.
+     *
+     * @param context {@link Context} for resolving resources
      */
-    public Stripe() { }
+    public Stripe(@NonNull Context context) {
```

Comment:
This is the breaking part of it -- there's no way for me to get a guid here without a context object, and we don't want to have it be skippable.